client-js/src/types.ts

```diff
@@ -1,13 +1,21 @@
-import { CLKeyParameters } from "casper-js-sdk";
+import { CLType, CLValue, CLKeyParameters } from "casper-js-sdk";
+
+export enum CEP47Events {
+  Mint = "Mint",
+  Transfer = "Transfer",
+  Burn = "Burn",
+  MetadataUpdate = "MetadataUpdate",
+  Approve = "Approve",
```

Comment:
I am a bit surprised to see a declaration of CEP47 events and not CES

```
Mint
Burn
Approval
ApprovalRevoked
ApprovalForAll
RevokedForAll
Transfer
MetadataUpdated (please not the ending d compared to CEP47)
VariablesSet
Migration
```

It seems to me unclear that we are slowly emphasing on using CES and not CEP47 which is a backward compatibility support but the focus is on CES events.

Reply:
this is because we're using external library for `CES` events handling so we don't need it in our codebase

Reply:
got that thanks